plugin/helpers_plugin.go

```diff
@@ -54,3 +54,24 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+// GetPluginAssetURL builds a URL to the given asset in the assets directory.
+func (p *HelpersImpl) GetPluginAssetURL(pluginID, asset string) (*url.URL, error) {
+	if len(pluginID) == 0 {
+		return nil, errors.New("empty pluginID provided")
+	}
+
+	if len(asset) == 0 {
+		return nil, errors.New("empty asset name provided")
+	}
+
+	siteURL := *p.API.GetConfig().ServiceSettings.SiteURL
+	if siteURL == "" {
+		return nil, errors.New("no SiteURL configured by the server")
+	}
```

Comment:
Nit: Add a newline to separate the blocks.
```suggestion
	}
	
```

Reply:
@hanzei thanks for the helpful feedback, I have implemented them.